### ISSUE TITLE
Expression editor does not show and an error shows in my logs.

### ISSUE THREAD

**Issue by vindolin** [grafana/grafana#63136]
**What happened**:

I get this error in my logs and the expression editor doesn't show up:
```
msg="Query data error" error="expression request error: failed to parse expression 'sad_expression': invalid math command type: expr: unexpected EOF in input: F()"
```

Complete line:
```
logger=context userId=1 orgId=1 uname=admin t=2023-02-08T19:40:25.303264+01:00 level=error msg="Query data error" error="expression request error: failed to parse expression 'sad_expression': invalid math command type: expr: unexpected EOF in input: F()" remote_addr=192.168.178.92 traceID=`
```

![image](https://user-images.githubusercontent.com/834347/217624103-a60da724-ce58-4330-9b0f-7861cb6a862e.png)

**What you expected to happen**:
The UI for the expression editor should open.

**How to reproduce it (as minimally and precisely as possible)**:
I click on the +Expression button in a graph editor window.

**Anything else we need to know?**:
This happens for every graph on my system.
I don't know when this started, I edited expressions some months ago.

**Environment**: 
- Grafana version: 9.3.6
- Data source type & version: Influxdb v1.8.10
- OS Grafana is installed on: Debian Buster
- User OS & Browser: Windows, Firefox/Chrome
- Grafana plugins: none
- Others:

**Comment by usmangt** [comment]
Hi @vindolin,

Thanks for opening this issue.

This seems to be a known issue as previously reported here: https://github.com/grafana/grafana/issues/62598


Please check it and let us know if that is the same as your case (or different)?

Thanks

**Comment by vindolin** [comment]
Thank you, yes it looks like it's the same problem and I see that a fix is targeted for 9.3.7 :)
